### PR TITLE
fix: icons for users, app users, and groups

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
## Describe your changes

* Fixed the users icon
* Fixed the copy for the application users icon
* Added Groups

## Checklist

- [ ] The first paragraph of the page is on one line.
- [ ] The other lines have a line break at 90 characters.
- [X] I checked the output.
- [ ] I applied the [style guide](../styleguide.md).
- [ ] My links start with `/docs/`.

### CODE REVIEW

File: src/components/ConsoleIcons/index.tsx [hunk 1]
```diff
@@ -132,7 +132,13 @@ export default function ConsoleLabel({name}): ReactElement {
     case 'users':
       return (
         <>
-          <ConsoleIconWrapper icon={ConsoleIcons.people} /> <b>Users</b>
+          <ConsoleIconWrapper icon={ConsoleIcons.user} /> <b>Users</b>
```
Comment (ArthurFlag) [inline src/components/ConsoleIcons/index.tsx:135]: That's not what I experience in the sidebar of any service?

Screenshot 2024-05-13 at 12 42 33

Current state seems to be correct for that area. Where did you see the `users` label connected to a single user icon? We might need to add a case.

**Review by ArthurFlag — CHANGES_REQUESTED** [review]

File: src/components/ConsoleIcons/index.tsx (hunk 1 above)
Comment (staceysalamon-aiven) [reply src/components/ConsoleIcons/index.tsx:135]: Ah, okay, so this is service users? Org users are managed in Admin. 
image

Comment (staceysalamon-aiven) [reply src/components/ConsoleIcons/index.tsx:135]: Speaking of which, I've just noticed as well that the view profile icon is different for org users versus applications users.

Comment (staceysalamon-aiven) [reply src/components/ConsoleIcons/index.tsx:142]: Added cases for these 👍

File: src/components/ConsoleIcons/index.tsx [hunk 2]
```diff
@@ -198,10 +218,11 @@ export default function ConsoleLabel({name}): ReactElement {
     case 'applicationusers':
       return (
         <>
-          <ConsoleIconWrapper icon={ConsoleIcons.appUsers} /> <b>Users</b>
+          <ConsoleIconWrapper icon={ConsoleIcons.appUsers} />{' '}
+          <b>Application users</b>
         </>
       );
-    case 'viewprofile':
+    case 'viewappuserprofile':
```
Comment (ArthurFlag) [inline src/components/ConsoleIcons/index.tsx:225]: Have you checked if it's used already? If so, the files using this case must be updated :)

Comment (staceysalamon-aiven) [reply src/components/ConsoleIcons/index.tsx:225]: Not from what I can see, unless I'm missing something?

File: src/components/ConsoleIcons/index.tsx (hunk 1 above)
Comment (staceysalamon-aiven) [reply src/components/ConsoleIcons/index.tsx:142]: Missed the service users page, but asking the team about this since it doesn't seem totally accurate?

**Pull request merged**